coalib/coala.py

```diff
@@ -71,10 +71,11 @@ def main():
     except BaseException as exception:  # pylint: disable=broad-except
         return get_exitcode(exception, log_printer)
 
-    if args.non_interactive:
+    if args.non_interactive and args.format:
+        return mode_non_interactive(console_printer, args), mode_format()
```

Comment:
shouldn't this rather throw an exception? We handle conflicts somewhere and this is a conflict, you can't just run it twice.